db/fixtures/miq_product_features.yml

```diff
@@ -1822,263 +1822,206 @@
   :identifier: rss
 
 # Policy
-- :name: Explorer
-  :description: Control Explorer
+- :name: Control
+  :description: Control
   :feature_type: node
-  :identifier: control_explorer
+  :identifier: control
   :children:
-  - :name: View All Records
-    :description: View all Records in Control Explorer
-    :feature_type: view
-    :hidden: true
-    :identifier: control_explorer_view
   - :name: Policy Profiles
-    :description: Everything under Policy Profiles Accordion
+    :description: Everything under Policy Profiles
     :protected: true
     :feature_type: node
-    :hidden: true
-    :identifier: policy_profile
+    :identifier: miq_policy_set
     :children:
     - :name: View
       :description: View Policy Profiles
       :feature_type: view
-      :identifier: policy_profile_view
+      :identifier: miq_policy_set_view
     - :name: Modify
       :description: Modify Policy Profiles
       :feature_type: admin
-      :hidden: true
-      :identifier: policy_profile_admin
+      :identifier: miq_policy_set_admin
       :children:
       - :name: Add
         :description: Add a Policy Profile
         :feature_type: admin
-        :hidden: true
-        :identifier: profile_new
+        :identifier: miq_policy_set_new
       - :name: Delete
         :description: Delete a Policy Profile
         :feature_type: admin
-        :hidden: true
-        :identifier: profile_delete
+        :identifier: miq_policy_set_delete
       - :name: Edit
         :description: Edit a Policy Profile
         :feature_type: admin
-        :hidden: true
-        :identifier: profile_edit
-      - :name: Edit Policies Assignment
```

Comment:
@h-kataria Is there a reason this was deleted?  This is still in the API, so I need to determine if this was something that was supposed to be removed (and why), or if this needs to be added back in.

I don't see anything in the PR description or documentation in an associated PR.

cc @gtanzillo

Reply:
Okay, this might be fine for the API.  Might not be used.